workers/unity/Packages/io.improbable.gdk.tools/GdkToolsConfigurationWindow.cs

```diff
@@ -172,14 +172,15 @@ private void DrawCustomSnapshotDir()
             using (new EditorGUILayout.HorizontalScope())
             {
                 EditorGUI.BeginDisabledGroup(true);
-                    EditorGUILayout.TextField(CustomSnapshotPathLabel, toolsConfig.CustomSnapshotPath);
+                EditorGUILayout.TextField(CustomSnapshotPathLabel, PlayerPrefs.GetString("CustomSnapshotPath", "snapshots/default.snapshot"));
```

Comment:
The PlayerPrefs should be abstracted away, see https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/io.improbable.gdk.mobile/Editor/MobileLaunchConfig.cs for an example.